communicator/ssh/communicator.go

```diff
@@ -278,6 +282,22 @@ func (c *Communicator) Start(cmd *remote.Cmd) error {
 		return err
 	}
 
+	// Start a goroutine that executes every KeepAliveInterval to keep the session alive
+	// will return when the session no longer responds to requests
+	go func() {
+		if c.config.KeepAliveInterval < 0 {
```

Comment:
This will panics below with a timeout of `0`, and should probably be limited to a reasonable minimum like 1 second, so we don't flood the connection with `1ns` keepalives.